Components/Cards/cards.css

```diff
@@ -0,0 +1,45 @@
+/* cards */
+.cart-versions {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.cart {
+    margin: 1rem;
+    padding: 1rem;
+    max-width: 100%;
+    font-size: 10px;
+}
+
+.img-contents {
+    background: white;
+    width: 100%;
+    max-width: 220px;
+    color: black;
+}
+
+.img-size {
+    width: 100%;
+}
+
+.rs99 {
```

Comment:
You can rename this to `.line-through` or `price-slashed` for usability/readability.